### ISSUE TITLE
Add Info Window

### ISSUE THREAD

**Issue by yankeeinlondon** [inocan-group/vue3-google-map#38]
Adding the **Info Windows** feature to this repo is a _feature-request_ for this library. We are happy to accept PR's for this work as there is not immediate demand internally for this feature but we do understand why it might be useful and it will eventually be incorporated.

**Comment by web-programmer-here** [comment]
+1 for this feature too, it is going to be very useful

**Comment by yankeeinlondon** [comment]
We're not currently using this functionality so it's not getting a ton of attention but would you consider adding in a pull request? Probably the fastest way to getting this in.

**Comment by HusamIbrahim** [comment]
Added in `v0.8.4`. Can be used like this:
```
<InfoWindow :options="{ position, content: 'Your content here' }" />
```

Or like this:
```
<InfoWindow :options="{ position }">
Your content here
</InfoWindow>
```

Documentation to be added soon

**Comment by Baroshem** [comment]
Tested it and it works great. I have one question @HusamIbrahim. Could you tell me how can I nest a info window inside of a marker so that when I click on a marker it will open an info window?

**Comment by eskiesirius** [comment]
may i see the screenshot of this? i cant make this work

**Comment by HusamIbrahim** [comment]
> may i see the screenshot of this? i cant make this work

Please share a reproduction @eskiesirius. You can use https://vite.new/vue.
In any case documentation will be added soon.

**Comment by HusamIbrahim** [comment]
@Baroshem Added nesting in `v0.9.0`. Basic syntax is:

```
<Marker :options="{ position }">
  <InfoWindow>Your content here</InfoWindow>
</Marker>
```

**Comment by Baroshem** [comment]
@HusamIbrahim This is just great!

Yesterday I managed to create a workaround for the current code to utilize the marker and infowindow like following:

```html
  <div>
    <google-marker :options="options" @click="open = !open"/>
    <info-window
      v-if="popup && open"
      :options="{ ...options, content: popup }"
      @closeclick="open = !open"
    />
  </div>
```

And today you just added a nested functionality that works even better. 

I can now utilize the full functionality. Thank you very much!

I am really happy that I have selected your library instead of others :)
